Remove Homebrew cruft and let MacPorts take care of the -arch flags

--- CMakeLists.txt.orig
+++ CMakeLists.txt
@@ -62,24 +62,6 @@
     include(cmake/XcodeHelper.cmake)
 endif()
 
-if(APPLE AND NOT IOS)
-    if(${CMAKE_SYSTEM_PROCESSOR} STREQUAL "arm")
-	set(ARCHFLAGS "arm64")
-	list(APPEND CMAKE_PREFIX_PATH /opt/homebrew/opt)
-	include_directories(/opt/homebrew/opt/openssl@1.1/include/)
-	include_directories(/opt/homebrew/opt/openssl@1.1/include/openssl/)
-	include_directories(/opt/homebrew/opt/freetype/include/freetype2/)
-	include_directories(/opt/homebrew/opt/harfbuzz/include/harfbuzz/)
-    else()
-	set(ARCHFLAGS "${CMAKE_SYSTEM_PROCESSOR}")
-	list(APPEND CMAKE_PREFIX_PATH /usr/local/opt)
-	include_directories(/usr/local/opt/openssl@1.1/include/)
-	include_directories(/usr/local/opt/openssl@1.1/include/openssl/)
-	include_directories(/usr/local/opt/freetype/include/freetype2/)
-	include_directories(/usr/local/opt/harfbuzz/include/harfbuzz/)
-    endif()
-endif()
-
 if (IOS)
     ADD_DEFINITIONS(-DMOBILE_STK)
     ADD_DEFINITIONS(-DIOS_STK)
@@ -356,10 +338,7 @@
 # Set include paths
 include_directories(${STK_SOURCE_DIR})
 
-if(APPLE AND NOT IOS)
-    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -arch ${ARCHFLAGS}")
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -arch ${ARCHFLAGS} -F/Library/Frameworks")
-elseif(MSVC)
+if(MSVC)
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /MP")   # Enable multi-processor compilation (faster)
 endif()
 
--- lib/bullet/CMakeLists.txt.orig
+++ lib/bullet/CMakeLists.txt
@@ -2,11 +2,6 @@
 
 include_directories(${CMAKE_CURRENT_SOURCE_DIR}/src/)
 
-if(APPLE AND NOT IOS)
-	set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -arch ${ARCHFLAGS}")
-	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -arch ${ARCHFLAGS} -F/Library/Frameworks")
-endif()
-
 # libbulletmath
 add_library(bulletmath STATIC
 	src/LinearMath/btAabbUtil2.h           src/LinearMath/btMotionState.h
--- lib/enet/CMakeLists.txt.orig
+++ lib/enet/CMakeLists.txt
@@ -28,11 +28,6 @@
 	add_definitions(-Wno-error)
 endif()
 
-if(APPLE AND NOT IOS)
-	set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -arch ${ARCHFLAGS}")
-	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -arch ${ARCHFLAGS} -F/Library/Frameworks")
-endif()
-
 if(HAS_FCNTL)
     add_definitions(-DHAS_FCNTL=1)
 endif()
--- lib/irrlicht/CMakeLists.txt.orig
+++ lib/irrlicht/CMakeLists.txt
@@ -32,8 +32,6 @@
 
 if(APPLE AND NOT IOS)
     include_directories("${CMAKE_CURRENT_SOURCE_DIR}/source/Irrlicht/MacOSX" "${CMAKE_CURRENT_SOURCE_DIR}/source/Irrlicht")
-    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -arch ${ARCHFLAGS}")
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -arch ${ARCHFLAGS} -F/Library/Frameworks")
 endif()
 
 add_definitions(-DNDEBUG=1 -DIRRLICHT_EXPORTS=1 -DPNG_THREAD_UNSAFE_OK -DPNG_NO_MMX_CODE -DPNG_NO_MNG_FEATURES)
